Source/DafnyCompilers/Interp.dfy

```diff
@@ -272,18 +272,18 @@ module Interp {
       case Apply(Eager(op), args: seq<Expr>) =>
         var Return(argvs, ctx) :- InterpExprs(args, env, ctx);
         LiftPureResult(ctx, match op {
-            case UnaryOp(op: UnaryOp) =>
-              InterpUnaryOp(e, op, argvs[0])
-            case BinaryOp(bop: BinaryOp) =>
-              assert !bop.BV? && !bop.Datatypes?;
-              InterpBinaryOp(e, bop, argvs[0], argvs[1])
-            case TernaryOp(top: TernaryOp) =>
-              InterpTernaryOp(e, top, argvs[0], argvs[1], argvs[2])
-            case Builtin(Display(ty)) =>
-              InterpDisplay(e, ty.kind, argvs)
-            case FunctionCall() =>
-              InterpFunctionCall(e, env, argvs[0], argvs[1..])
-          })
+                         case UnaryOp(op: UnaryOp) =>
```

Comment:
there's a code highlighting bug

Reply:
With the newest version, the code diff is no longer a problem, can you have a look and let me know if this is ok?
I still think the indentation of the match case should be more aligned.